cfgov/unprocessed/apps/teachers-digital-platform/js/modals.js

```diff
@@ -111,7 +110,7 @@ function init() {
 function handleClicks() {
   document.addEventListener('click', (event) => {
     const t = event.target;
-    const opener = closest(t, '[data-open-modal]');
+    const opener = t.closest('[data-open-modal]');
```

Comment:
You could update `t` to `target` if you want. There are few references further down too.